views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -541,20 +549,29 @@ define([
                     equal:  { label: '=',           latex: '=',         fn: 'write',    desc: 'Equal'},
                     lower:  { label: '<',           latex: '<',         fn: 'write',    desc: 'Lower than'},
                     greater: { label: '>',          latex: '>',         fn: 'write',    desc: 'Greater than'},
-                    subscript: { label: 'x&#8345;', latex: '_',         fn: 'cmd',      desc: 'Subscript'},
-                    lbrace: { label: '{',           latex: '{',   fn: 'cmd',      desc: 'Left brace/curly bracket'},
-                    rbrace: { label: '}',           latex: '}',  fn: 'cmd',      desc: 'Right brace/curly bracket'},
-                    lparen: { label: '(',           latex: '(',   fn: 'cmd',      desc: 'Left parenthese/round bracket'},
-                    rparen: { label: ')',           latex: ')',  fn: 'cmd',      desc: 'Right parenthese/round bracket'},
-                    integral: { label: '&#x222b;',  latex: '\\int',     fn: 'cmd',      desc: 'Indefinitve integral'},
+                    subscript: { label: 'x&#8336;', latex: '_',         fn: 'cmd',      desc: 'Subscript'},
+                    lbrace: { label: '{',           latex: '{',         fn: 'cmd',      desc: 'Left brace/curly bracket'},
+                    rbrace: { label: '}',           latex: '}',         fn: 'cmd',      desc: 'Right brace/curly bracket'},
+                    lparen: { label: '(',           latex: '(',         fn: 'write',    desc: 'Left parenthesis/round bracket'},
```

Comment:
Its `parentheses`

Reply:
Now I'm confused :)
https://github.com/oat-sa/extension-tao-itemqti-pci/pull/196#discussion_r423272513 
Additional resource https://english.stackexchange.com/questions/130219/parentheses-vs-parenthesis

Reply:
@oatymart can you please confirm if its parentheses or parenthesis

Reply:
parenthesis - single
parentheses - plural